src/rspamd.h

```diff
@@ -308,7 +308,7 @@ struct rspamd_main {
 	rspamd_logger_t *logger;
 	uid_t workers_uid;                                          /**< worker's uid running to                        */
 	gid_t workers_gid;                                          /**< worker's gid running to						*/
-	gboolean is_privilleged;                                    /**< true if run in privilleged mode                */
+	gboolean is_privileged;                                    /**< true if run in privileged mode                */
```

Comment:
I should probably adjust the whitespace here (the trailing side seems to already be mess)